meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -7,7 +7,6 @@ PREFERRED_PROVIDER_virtual/bootloader ?=  "u-boot"
 PREFERRED_VERSION_u-boot ?= "2018.09"
 PREFERRED_VERSION_u-boot-fw-utils ?= "2018.09"
 PREFERRED_VERSION_pulseaudio ?= "11.1%"
-PREFERRED_VERSION_openssl ?= "1.1.0%"
```

Comment:
I think this one needs a closer and a separate look - as this should have nothing to do with the glibc problems other patches in the PR solve. And downgrading the OpenSSL doesn't look good to me. Let me try to look into this while I'm prepraring a build for #50.

Reply:
No, apparently something depends on openssl 1.0.2p. Which it didn't before.

Reply:
Yeah, I understood from your discussion in #56 that's something unexpected and possibly related to sumo updates, however it's a bit hard to believe they'd change something in favor of downgrading the OpenSSL, so I think that may be some bug/misconfig somewhere we could chase.

Reply:
So what do think? 

BTW if we can pull this then we have a more or less stable sumo. The new thud branch that I will send a PR for next has openssl 1.1 really. And U-Boot 2019.01, kernel 4.20. Is much more interresting :-)

Reply:
Just got to investigate that - right, what you've outline is exactly the reason. Let's then just go with removing the preferred version indeed for now and see if `thud`'s packages no longer require 1.0. 1.1.1 is now the LTS version and has good security-related updates across the board, so I'd be interested in updating to that, but that's not really a critical piece for our image anyway :)

Reply:
Yes thud has 1.1.1a see https://github.com/htot/meta-intel-edison/blob/thud/docs/edison-image-edison.manifest